packages/integration-tests/webpack.config.ts

```diff
@@ -0,0 +1,34 @@
+import { Configuration } from 'webpack';
+
+const config = function(userConfig: Record<string, unknown>): Configuration {
+  return {
+    ...userConfig,
+    mode: 'none',
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          exclude: /node_modules/,
+          loader: 'babel-loader',
+        },
+        {
+          test: /\.ts$/,
+          exclude: /node_modules/,
+          loader: 'ts-loader',
+          options: {
+            // This has a big impact on test build speed.
+            transpileOnly: true,
```

Comment:
I see this option documented in https://webpack.js.org/guides/build-performance/#typescript-loader

I could not find where we're actually doing the type-checking?

Reply:
Added a non-compiled type checking script and removed `ts-loader` from webpack which we don't need anymore.